models/models.go

```diff
@@ -122,3 +122,11 @@ type StepRunResultsModel struct {
 	ErrorStr string                      `json:"error_str" yaml:"error_str"`
 	ExitCode int                         `json:"exit_code" yaml:"exit_code"`
 }
+
+// TestResultStepInfo ...
```

Comment:
What is a `TestResultStepInfo` object? What is it's purpose within the system? Let us write meaningful comments from now on.